web/client/translations/data.hr-HR.json

```diff
@@ -1153,6 +1153,7 @@
             "showPreview": "Prikaži pretpregled",
             "advancedSettings": "Napredne mogućnosti",
             "templateMetadataAvailable": "Metapodaci distupni iz Dublin Core formata: abstract, boundingBox, contributor, creator, description, format, identifier, references, rights, source, subject, temporal, title, type, uri",
+            "invalidUrlHttpProtocol": "URL se ne podudara s protokolom MapStore HTTP {protocol}",
```

Comment:
We usually translate only the default languages supported by mapstore DE, US, ES, FR and IT so this could be removed